frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -12,7 +12,6 @@
     <span class="text-uppercase mb-1 checkout-confirm-th"><%= Spree.t(:total) %></span>
     <% @order.line_items.each do |item| %>
       <div class="checkout-confirm-product">
-        <%= link_to plp_image(item.variant.product, ''), item.variant.product %>
```

Comment:
@aplegatt why do we want to  remove this?

Reply:
Must have deleted it by mistake 😞